install.md

```diff
@@ -7,6 +7,34 @@ main_link: True
 
 # Install or upgrade
 
+## mozregression-gui
+
+For the graphical interface, we want to provide a simple installation process:
+
+- **Windows**:
+  Just download the latest **mozregression-gui.exe** file from the
+  [github releases]. This is an installer, so once the file is downloaded
```

Comment:
I don't think we need to specify that it's an installer.

This is an installer, so once ... -> Once ...